spec/configuration_store_spec.lua

```diff
@@ -151,20 +151,24 @@ describe('Configuration Store', function()
 
     it('returns stale records by default', function()
       local store = configuration.new()
-      local service =  { id = '21', hosts = { 'example.com', 'localhost' } }
+      local service =  { id = '21', hosts = { 'example.com' } }
 
-      store:add(service, -1)
+      -- Can't add stale info with config.add. Need to do it through the
+      -- internals of the object.
+      store.cache:set('example.com', { service }, -1)
 
       assert.same({ service }, store:find_by_host('example.com'))
     end)
 
     it('does not return stale records when disabled', function()
       local store = configuration.new()
-      local service =  { id = '21', hosts = { 'example.com', 'localhost' } }
+      local service =  { id = '21', hosts = { 'example.com' } }
 
-      store:add(service, -1)
+      -- Can't add stale info with config.add. Need to do it through the
```

Comment:
We could add one with expiry 0 ?

Reply:
I just tried it and it does not work. I guess it needs a bit of time to delete it even with ttl=0.